src/uri.rs

```diff
@@ -380,6 +380,17 @@ impl UUri {
         }
     }
 
+    /// Gets a URI that consists of wildcards and the specific resource ID.
+    pub fn any_with_resource_id(resource_id: u32) -> Self {
```

Comment:
how about changing the implementation of `Self::any` to
```rust
Self::any_with_resource_id(WILDCARD_RESOURCE_ID)
```
in order to reduce code duplication?

Reply:
Good idea.